config/checkstyle_checks.xml

```diff
@@ -399,6 +399,12 @@
       //LITERAL_CATCH//METHOD_CALL[.//IDENT[@text = 'printStackTrace']]/.."/>
       <message key="matchxpath.match" value="Avoid using 'printStackTrace'."/>
     </module>
+    <module name="MatchXpath">
+      <property name="query" value="//METHOD_DEF/MODIFIERS//
+            ANNOTATION[.//IDENT[@text='Test']]/ANNOTATION_MEMBER_VALUE_PAIR
```

Comment:
Sorry I missed this. I would also look at changing `.//IDENT` to `./IDENT`.

Reply:
```
➜  src cat config.xml 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
        "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
    <property name="charset" value="UTF-8"/>

    <module name="TreeWalker">
    <module name="MatchXpath">
      <property name="query" value="//METHOD_DEF/MODIFIERS//
            ANNOTATION[./IDENT[@text='Test']]/ANNOTATION_MEMBER_VALUE_PAIR
            [./IDENT[@text='expected']]"/>
      <message key="matchxpath.match" value="Avoid using ''expected'' in Test annotation."/>
    </module>
    </module>
</module>
➜  src java -jar /home/nick/IdeaProjects/checkstyle/target/checkstyle-8.40-SNAPSHOT-all.jar -c config.xml XPathExpectedMatcher.java
Starting audit...
[ERROR] /home/nick/Desktop/Tester/src/XPathExpectedMatcher.java:4:11: Avoid using 'expected' in Test annotation. [MatchXpath]
Audit done.
Checkstyle ends with 1 errors.

```
Fixed.